PIL/Image.py

```diff
@@ -588,11 +588,11 @@ def _dump(self, file=None, format=None):
 
         self.load()
         if not format or format == "PPM":
-            self.im.save_ppm(file)
+            self.im.save_ppm(file, **options)
```

Comment:
I doubt that. Both production (as installed by distros) and dev code uses `_dump`. Besides, debug code uses an entirely different code path is confusing.